src/Libraries/DSOfficeUtilities/Excel.cs

```diff
@@ -105,7 +105,7 @@ public static bool ExcelProcessRunning
         {
             get
             {
-                return Process.GetProcessesByName("EXCEL").Length != 0;
+                return GetExcelProcess(_app) != null;
```

Comment:
could you briefly explain these changes?

Reply:
With my change we can check if the Excel process that is associated with the COM object "_app" is alive.
Previously we were checking if any process called "EXCEL" was running. It seemed that could get into situations where multiple Excel processes could get mixed up